### PR TITLE
[Forums] Added new forum message window [Open in a new window]

### PR DESCRIPTION
**Pull request by defnax**
* Added new forum message window [Open in a new window]

### CODE REVIEW

File: retroshare-gui/src/retroshare-gui.pro [hunk 1]
```diff
@@ -1322,6 +1324,7 @@ gxsforums {
 		gui/gxsforums/GxsForumThreadWidget.cpp \
 		gui/gxsforums/GxsForumModel.cpp \
 		gui/gxsforums/GxsForumUserNotify.cpp \
+		gui/gxsforums/ForumMessageView.cpp \
```
Comment (PhenomRetroShare) [inline retroshare-gui/src/retroshare-gui.pro]: Why didn't you named it GxsForumMessageView or best GxsForumMessageDialog ?

Comment (defnax) [reply retroshare-gui/src/retroshare-gui.pro]: i can rename it, but i never like add naming with GXS before , normaly on gui we doesnt need everything naming with gxs? only to look same when other classes starts with gxs right..?

Comment (PhenomRetroShare) [reply retroshare-gui/src/retroshare-gui.pro]: Neither do I, but all other class are named like this so keep homogeneity.

Comment (defnax) [reply retroshare-gui/src/retroshare-gui.pro]: ok

Comment (defnax) [reply retroshare-gui/src/retroshare-gui.pro]: > Neither do I, but all other class are named like this so keep homogeneity.

hi i has renamed it, if you see some other things?

Comment (PhenomRetroShare) [reply retroshare-gui/src/retroshare-gui.pro]: Yes some little, I'll comment them latter. I don't have the time now.

**Pull request closed without merging**